.bazelrc

```diff
@@ -163,6 +163,8 @@ common --nolegacy_important_outputs
 
 # Use a static PATH variable to prevent unnecessary rebuilds of dependencies like protobuf.
 common --incompatible_strict_action_env
+common --action_env=DEVELOPER_DIR
+common --host_action_env=DEVELOPER_DIR
```

Comment:
I think this could be set for MacOS only to reduce the risk of cache poisoning to only MacOS builds.

I wonder if we should set the default value `/Library/Developer/CommandLineTools` here as well. To override the default, the user could set their own `--action_env=DEVELOPER_DIR=<custom_value>`.

WDYT?

Reply:
> I think this could be set for MacOS only to reduce the risk of cache poisoning to only MacOS builds.

Agree. I missed it during my refactoring of stuff.

> I wonder if we should set the default value /Library/Developer/CommandLineTools here as well. To override the default, the user could set their own `--action_env=DEVELOPER_DIR=<custom_value>`

It's possible to not have the command-line tools installed. We want to be able to support people that only have Xcode installed. So I think how I have it is fine.